### PR TITLE
Change "Wiimote" to "Wii Remote" in Interface

### PR DESCRIPTION
**Pull request by Pringo**
The usage of "Wii Remote" and "Wiimote" in the interface is inconsistent. "Wiimote" is also not a real word nor is it an official product name. Therefore I have changed instances of "Wiimote" in the UI to instead say "Wii Remote". I also made a couple of minor grammatical changes as well.

---
This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/4338)

### CODE REVIEW

**Comment by mbc07** [comment]
This PR now matches the wording we've been using on the wiki for years! Sweet!

**Comment by MaJoRoesch** [comment]
Oh no! The nitpickers have struck! :P 

Ok ok, this is well needed, since wiimote is kind of silly. But we'll keep using wiimote on the blog to keep it alive! (actually to mix it up so we don't keep saying Wii Remote over and over, same thing!)

**Comment by Helios747** [comment]
Can you fix linter please?

Delroth might have updated the buildbot to LLVM 3.9 so clang-format may be giving you different results. I'm not sure if he did. What version are you running of clang-format?

**Comment by SeannyM** [comment]
The Android UI also uses "Wiimote", you might want to change that too.

File: Source/Core/DolphinWX/FrameTools.cpp [hunk 1]
```diff
@@ -1538,7 +1538,7 @@ void CFrame::ConnectWiimote(int wm_idx, bool connect)
   {
     bool was_unpaused = Core::PauseAndLock(true);
     GetUsbPointer()->AccessWiiMote(wm_idx | 0x100)->Activate(connect);
-    const char* message = connect ? "Wiimote %i connected" : "Wiimote %i disconnected";
+    const char* message = connect ? "Wiimote %i connected" : "Wii Remote %i disconnected";
```
Comment (leoetlino) [inline Source/Core/DolphinWX/FrameTools.cpp]: Wiimote %i connected => Wii Remote %i connected for consistency

File: Source/Core/DolphinWX/ControllerConfigDiag.cpp [hunk 2]
```diff
@@ -567,7 +567,7 @@ void ControllerConfigDiag::SaveWiimoteSource()
 
   for (unsigned int i = 0; i < MAX_WIIMOTES; ++i)
   {
-    std::string secname("Wiimote");
+    std::string secname("Wii Remote");
```
Comment (JosJuice) [inline Source/Core/DolphinWX/ControllerConfigDiag.cpp]: Yes. Let's only change the user-visible strings, not the strings that are used in INIs.

File: Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp [hunk 3]
```diff
@@ -312,7 +312,7 @@ Wiimote::Wiimote(const unsigned int index)
 
 std::string Wiimote::GetName() const
 {
-  return std::string("Wiimote") + char('1' + m_index);
+  return std::string("Wii Remote") + char('1' + m_index);
```
Comment (JosJuice) [inline Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp]: This one may be a config string. (I'm not sure.) This needs to be checked before merging.

**Comment by JosJuice** [comment]
I like this change (except for the config breaking, but that should be easy to fix).

**Comment by Pringo** [comment]
@SeannyM I agree that the wording should be changed in the Android version as well but I am not very familiar with the Android version interface. It might be better if someone else took care of that.

**Comment by SeannyM** [comment]
@Pringo all the Android strings are stored in one file: https://github.com/dolphin-emu/dolphin/blob/master/Source/Android/app/src/main/res/values/strings.xml
All you have to do is change them from here ;)

**Comment by Pringo** [comment]
@SeannyM Okay. Do you have any suggestions what "Wii Controllers (Wiimotes)" should be changed to? I'm not sure what that screen is for and what would make sense there. I was thinking of just deleting the parenthesis part if that works.

**Comment by Helios747** [comment]
@Pringo 

Now, unless delroth updated clang to 3.9 (the latest stable), the buildbot is still on 3.8, so go download LLVM 3.8 for Windows here

http://llvm.org/releases/download.html

then check out the first section of the intro in contributing.md

https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md

**Comment by SeannyM** [comment]
@Pringo I actually think that's an unused string, but you can just remote the "(Wiimotes)" part for now for consistency.

**Comment by Pringo** [comment]
Okay, I think I got everything fixed now.

**Comment by Helios747** [comment]
LGTM

**Comment by leoetlino** [comment]
This will need a rebase.

**Comment by Pringo** [comment]
I will try to fix this later.

**Pull request closed without merging**